doc/lsstDebug/index.rst

```diff
@@ -77,6 +77,10 @@ while importing `debug.py` will show:
 
 The ``pipetask run`` command line interface supports a flag ``--debug`` to import ``debug.py`` from your ``PYTHONPATH``.
 
+To use lsstDebug when running tests with pytest, put the ``debug.py`` file described above in your package's ``tests/`` directory and ``import debug`` in the test file you are running.
```

Comment:
Does it have to be in the `tests/` directory? I've never had to do that; it being on `PYTHONPATH` has always been sufficient.